cmd/osquery-perf/agent.go

```diff
@@ -412,16 +415,52 @@ func (a *agent) HostUsersMacOS() []fleet.HostUser {
 	return users
 }
 
+func extract(src, dst string) {
```

Comment:
Here I'm reusing the software fixtures from https://github.com/fleetdm/fleet/tree/main/server/vulnerabilities/testdata